src/Reflection/ReflectionClass.php

```diff
@@ -79,12 +79,15 @@ public static function export($className = null)
 
     public function __toString()
     {
-        $format  = "Class [ <user> class %s%s%s ] {\n";
+        $isObject = $this instanceof ReflectionObject;
+
+        $format  = "%s [ <user> class %s%s%s ] {\n";
         $format .= "  @@ %s %d-%d\n\n";
         $format .= "  - Constants [%d] {%s\n  }\n\n";
         $format .= "  - Static properties [%d] {%s\n  }\n\n";
         $format .= "  - Static methods [%d] {%s\n  }\n\n";
         $format .= "  - Properties [%d] {%s\n  }\n\n";
+        $format .= ($isObject ? "  - Dynamic properties [%d] {%s\n  }\n\n" : '%s%s');
```

Comment:
The parallelism between line 90 and line [153~154](https://github.com/Roave/BetterReflection/pull/92/files#diff-0b62f5919f325c15e41b6b4c09690ff7R151) makes this quite hard to follow. Maybe just hardcode the values in this particular line?